src/main/scala/com/johnsnowlabs/nlp/pretrained/PretrainedPipeline.scala

```diff
@@ -0,0 +1,29 @@
+package com.johnsnowlabs.nlp.pretrained
+
+import com.johnsnowlabs.nlp.LightPipeline
+import org.apache.spark.ml.PipelineModel
+import org.apache.spark.sql.DataFrame
+
+case class PretrainedPipeline(
+                               downloadName: String,
+                               language: String = "en",
+                               source: String = ResourceDownloader.publicLoc
```

Comment:
Shoot! I can see we have a filed here called `source` which is about the location of the model. If we decide to add a field to allow users to choose which model they want for one specific annotator, maybe we can choose `type` or something or rename this to `url` and use the `source`.

Reply:
right, agreed. I'll rename that parameter